test/salesforce/bulkdelete/main.go

```diff
@@ -0,0 +1,240 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/csv"
+	"fmt"
+	"io"
+	"log"
+	"log/slog"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/amp-labs/connectors/salesforce"
+	testUtils "github.com/amp-labs/connectors/test/utils"
+	"github.com/amp-labs/connectors/utils"
+)
+
+func main() {
+	handler := slog.NewTextHandler(os.Stdout, &slog.HandlerOptions{
+		Level: slog.LevelDebug,
+	})
+
+	logger := slog.New(handler)
+	slog.SetDefault(logger)
+
+	ctx := context.Background()
+
+	sfc, err := testUtils.Connector(ctx)
+	if err != nil {
+		slog.Error("Error creating Salesforce connector", "error", err)
+
+		return
+	}
+
+	defer func() {
+		_ = sfc.Close()
+	}()
+
+	// Prepare objects and write to Salesforce to be deleted
```

Comment:
Can you revise this section of comments to:
"We first create objects in Salesforce, and then we generate an in-memory CSV of the Salesforce IDs of the newly created objects, so that we can bulk-delete them."